test/test_fcontext.cpp

```diff
@@ -9,6 +9,7 @@
 #include <stdexcept>
 #include <string>
 #include <utility>
+#include <cstdio>
```

Comment:
I get an error without this include: *`sscanf` is not a member of namespace `std`*